src/sidebar/templates/top_bar.html

```diff
@@ -23,39 +23,39 @@
        The inner div is styled with 'content' to center it in
        the stream view.
   !-->
-  <div class="top-bar__inner content" ng-if="::isSidebar">
-    <group-list class="group-list" auth="auth"></group-list>
+  <div class="top-bar__inner content" ng-if="::vm.isSidebar">
+    <group-list class="group-list" auth="vm.auth"></group-list>
     <div class="top-bar__expander"></div>
     <a class="top-bar__apply-update-btn"
-       ng-if="pendingUpdateCount > 0"
-       ng-click="onApplyPendingUpdates()"
+       ng-if="vm.pendingUpdateCount > 0"
+       ng-click="vm.onApplyPendingUpdates()"
        h-tooltip
        tooltip-direction="up"
-       aria-label="Show {{pendingUpdateCount}} new/updated annotation(s)">
+       aria-label="Show {{vm.pendingUpdateCount}} new/updated annotation(s)">
        <svg-icon class="top-bar__apply-icon" name="'refresh'"></svg-icon>
     </a>
     <search-input
       class="search-input"
-      query="searchController.query()"
-      on-search="searchController.update($query)"
+      query="vm.searchController.query()"
+      on-search="vm.searchController.update($query)"
       title="Filter the annotation list">
     </search-input>
     <sort-dropdown
-      sort-keys-available="sortKeysAvailable"
-      sort-key="sortKey"
-      on-change-sort-key="onChangeSortKey({sortKey: sortKey})">
+      sort-keys-available="vm.sortKeysAvailable"
+      sort-key="vm.sortKey"
+      on-change-sort-key="vm.onChangeSortKey({sortKey: vm.sortKey})">
     </sort-dropdown>
     <a class="top-bar__btn"
-       ng-click="onSharePage()"
+       ng-click="vm.onSharePage()"
        title="Share this page">
       <i class="h-icon-annotation-share"></i>
     </a>
     <login-control
       class="login-control"
-      auth="auth"
+      auth="vm.auth"
       new-style="true"
-      on-show-help-panel="onShowHelpPanel()"
-      on-login="onLogin()"
+      on-show-help-panel="vm.onShowHelpPanel()"
+      on-login="vm.onLogin()"
       on-logout="onLogout()">
```

Comment:
🤕 - Gah, sloppy of me. I'll get this fixed and look into adding some tests or other form of checks.